### PR TITLE
Percona-mongo 0.2.0-3.4.13

### PR DESCRIPTION
**Pull request by timvaillancourt**
## Urgency
- [ ] Blocker 
- [X] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by timvaillancourt** [comment]
CC: @ryadav88

File: pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+1. Run `dcos percona-mongo pod restart mongo-rs-<NUM>`, e.g. `mongo-rs-2`.
+
+<a name="replacing-a-node"></a>
+## Replacing a Node
+
+This operation will move a node to a new system and will discard the persistent volumes at the prior system to be rebuilt at the new system. Perform this operation if a given system is about to be offlined or has already been offlined.
+
+**Note:** Nodes are not moved automatically. You must perform the following steps manually to move nodes to new systems. You can automate node replacement according to your own preferences.
+
+1. _ANY STEPS TO WIND DOWN A NODE BEFORE IT'S WIPED/DECOMMISSIONED GO HERE._
```
Comment (pavisandhu) [inline pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md]: This is placeholder text. Either add the relevant content, or remove this text.

File: pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+1. Run `dcos percona-mongo pod replace mongo-rs-<NUM>` to halt the current instance with id `<NUM>` (if still running) and launch a new instance elsewhere.
+
+For example, let's say `mongo-rs-2`'s host system has died and `mongo-rs-2` needs to be moved.
+
+1. _DETAILED INSTRUCTIONS FOR WINDING DOWN A NODE, IF NEEDED FOR YOUR SERVICE, GO HERE._
```
Comment (pavisandhu) [inline pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md]: This is placeholder text. Either add the relevant content, or remove this text.

File: pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md (hunk 1 above)
Comment (pavisandhu) [inline pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md]: This is placeholder text. Either add the relevant content, or remove this text.

File: pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md (hunk 2 above)
Comment (pavisandhu) [inline pages/services/percona-mongo/0.2.0-3.4.13/managing/index.md]: This is placeholder text. Either add the relevant content, or remove this text.

File: pages/services/percona-mongo/0.2.0-3.4.13/troubleshooting/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+You can also access the logs via the Mesos UI:
+1. Visit <dcos-url>/mesos to view the Mesos UI.
+1. Click the `Frameworks` tab in the upper left to get a list of services running in the cluster.
+1. Navigate into the correct framework for your needs. The scheduler runs under `marathon` with a task name matching the service name (default _`percona-mongo`_). Service nodes run under a framework whose name matches the service name (default _`percona-mongo`_).
+1. You should now see two lists of tasks. `Active Tasks` are tasks currently running, and `Completed Tasks` are tasks that have exited. Click on the `Sandbox` link for the task you wish to examine.
+1. The `Sandbox` view will list files named `stdout` and `stderr`. Click the file names to view the files in the browser, or click `Download` to download the file to your system for local examination. Note that very old tasks will have their Sandbox automatically deleted to limit disk space usage.
+
+<a name="accessing-metrics"></a>
+# Accessing Metrics
+_PROVIDE INSTRUCTIONS FOR ACCESSING METRICS OR INDICATE A LACK OF METRICS W/I LIMITATIONS.MD._
```
Comment (pavisandhu) [inline pages/services/percona-mongo/0.2.0-3.4.13/troubleshooting/index.md]: This is placeholder text. Either add the relevant content, or remove this text.

**Comment by timvaillancourt** [comment]
Thank you @pavisandhu. I have made these corrections in this PR now.

File: pages/services/percona-mongo/0.2.0-3.4.13/troubleshooting/index.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ You can also access the logs via the Mesos UI:
 
 <a name="accessing-metrics"></a>
 # Accessing Metrics
-_PROVIDE INSTRUCTIONS FOR ACCESSING METRICS OR INDICATE A LACK OF METRICS W/I LIMITATIONS.MD._
+Service-level metrics will be added in a future release of this service.
```
Comment (ToddGreenstein) [inline pages/services/percona-mongo/0.2.0-3.4.13/troubleshooting/index.md]: @timvaillancourt What does this mean?   What is the level of effort involved?

Comment (timvaillancourt) [reply pages/services/percona-mongo/0.2.0-3.4.13/troubleshooting/index.md]: @ToddGreenstein : We believe a medium-high amount of effort is involved and we have an alternative/workaround via the Percona PMM monitoring functionality supported in this release. I will update this page to clarify.

I believe the plan is to add DC/OS metrics in our next release.

**Pull request merged**